client/app/scripts/liveblog-edit/module.js

```diff
@@ -135,15 +141,24 @@ define([
             publish: function() {
                 $scope.actionPending = true;
                 notify.info(gettext('Saving post'));
+                var needsRemoval = $scope.currentPost && $scope.currentPost.sticky !== $scope.post.sticky;
                 postsService.savePost(blog._id,
                     $scope.currentPost,
                     getItemsFromEditor(),
-                    {post_status: 'open'}
+                    {post_status: 'open', sticky: $scope.post.sticky}
                 ).then(function(post) {
                     notify.pop();
                     notify.info(gettext('Post saved'));
+                    if (needsRemoval) {
```

Comment:
Why do we remove in the `publish` method ? :)
I think we already deal with deletion somewheere for update. Like when we publish a draft, we remove it from from the draft list. It would be awesome to trigger that at the same place